[
{"message":"file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237\n        throw new Error(y, { cause: u });\n              ^\n","level":"FATAL","type":"PLAYGROUND","test":"asyncify-boot","plugin":"compartilhe","details":"file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237\n        throw new Error(y, { cause: u });\n              ^\n","log":"/tmp/tmp.NPstW0xYD3"}
,
{"message":"Function get_usermeta is <strong>deprecated</strong> since version 3.0.0! Use get_user_meta() instead. in /wordpress/wp-includes/functions.php on line 6121","level":"DEPRECATED","type":"PHP","test":"asyncify-boot","plugin":"compartilhe","details":"[17-Apr-2025 03:59:30 UTC] PHP Deprecated:  Function get_usermeta is <strong>deprecated</strong> since version 3.0.0! Use get_user_meta() instead. in /wordpress/wp-includes/functions.php on line 6121","log":"/logs/plugins/c/compartilhe/error.json"}
,
{"message":"Undefined array key \"cred_login\" in /wordpress/wp-content/plugins/cred/cred.php on line 36","level":"WARNING","type":"PHP","test":"asyncify-boot","plugin":"compartilhe","details":"[17-Apr-2025 03:59:30 UTC] PHP Warning:  Undefined array key \"cred_login\" in /wordpress/wp-content/plugins/cred/cred.php on line 36","log":"/logs/plugins/c/compartilhe/error.json"}
,
{"message":"Function get_usermeta is <strong>deprecated</strong> since version 3.0.0! Use get_user_meta() instead. in /wordpress/wp-includes/functions.php on line 6121","level":"DEPRECATED","type":"PHP","test":"asyncify-boot","plugin":"compartilhe","details":"[17-Apr-2025 03:59:40 UTC] PHP Deprecated:  Function get_usermeta is <strong>deprecated</strong> since version 3.0.0! Use get_user_meta() instead. in /wordpress/wp-includes/functions.php on line 6121","log":"/logs/plugins/c/compartilhe/error.json"}
,
{"message":"Undefined array key \"cred_login\" in /wordpress/wp-content/plugins/cred/cred.php on line 36","level":"WARNING","type":"PHP","test":"asyncify-boot","plugin":"compartilhe","details":"[17-Apr-2025 03:59:40 UTC] PHP Warning:  Undefined array key \"cred_login\" in /wordpress/wp-content/plugins/cred/cred.php on line 36","log":"/logs/plugins/c/compartilhe/error.json"}
,
{"message":"Function get_usermeta is <strong>deprecated</strong> since version 3.0.0! Use get_user_meta() instead. in /wordpress/wp-includes/functions.php on line 6121","level":"DEPRECATED","type":"PHP","test":"asyncify-boot","plugin":"compartilhe","details":"[17-Apr-2025 03:59:49 UTC] PHP Deprecated:  Function get_usermeta is <strong>deprecated</strong> since version 3.0.0! Use get_user_meta() instead. in /wordpress/wp-includes/functions.php on line 6121","log":"/logs/plugins/c/compartilhe/error.json"}
,
{"message":"Undefined array key \"cred_login\" in /wordpress/wp-content/plugins/cred/cred.php on line 36","level":"WARNING","type":"PHP","test":"asyncify-boot","plugin":"compartilhe","details":"[17-Apr-2025 03:59:49 UTC] PHP Warning:  Undefined array key \"cred_login\" in /wordpress/wp-content/plugins/cred/cred.php on line 36","log":"/logs/plugins/c/compartilhe/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 duplicate column name: add_time.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"compartilhe","details":"[17-Apr-2025 04:00:56 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE `wp_cptch_whitelist` ADD `add_time` DATETIME;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_cptch_whitelist, `ADD_TIME`, datetime</li>\n<li>Executing: ALTER TABLE wp_cptch_whitelist ADD `add_time` text; | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 duplicate column name: add_time.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/captcha/captcha.php(256): WP_SQLite_DB->query('ALTER TABLE `wp...')\n#2 /wordpress/wp-content/plugins/captcha/captcha.php(306): cptch_create_table()\n#3 /wordpress/wp-includes/class-wp-hook.php(324): cptch_plugin_activate(false)\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_captch...', false)\n#7 /internal/eval.php(20): activate_plugin('captcha/captcha...')\n#8 {main}</pre>\n for query ALTER TABLE `wp_cptch_whitelist` ADD `add_time` DATETIME; made by activate_plugin, do_action('activate_captcha/captcha.php'), WP_Hook->do_action, WP_Hook->apply_filters, cptch_plugin_activate, cptch_create_table, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/c/compartilhe/error.json"}
,
{"message":"Unknown show type: KEYS FROM.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"compartilhe","details":"[17-Apr-2025 04:00:56 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>SHOW KEYS FROM `wp_cptch_whitelist` WHERE Key_name='ip'</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown show type: KEYS FROM.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/captcha/captcha.php(258): WP_SQLite_DB->query('SHOW KEYS FROM ...')\n#2 /wordpress/wp-content/plugins/captcha/captcha.php(306): cptch_create_table()\n#3 /wordpress/wp-includes/class-wp-hook.php(324): cptch_plugin_activate(false)\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_captch...', false)\n#7 /internal/eval.php(20): activate_plugin('captcha/captcha...')\n#8 {main}</pre>\n for query SHOW KEYS FROM `wp_cptch_whitelist` WHERE Key_name='ip' made by activate_plugin, do_action('activate_captcha/captcha.php'), WP_Hook->do_action, WP_Hook->apply_filters, cptch_plugin_activate, cptch_create_table, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/c/compartilhe/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 near \")\": syntax error.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"compartilhe","details":"[17-Apr-2025 04:00:56 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE `wp_cptch_whitelist` ADD UNIQUE(`ip`);</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_cptch_whitelist, wp_cptch_whitelist__(, UNIQUE</li>\n<li>Executing: CREATE UNIQUE INDEX &quot;wp_cptch_whitelist__(&quot; ON &quot;wp_cptch_whitelist&quot; (); | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 near \")\": syntax error.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/captcha/captcha.php(259): WP_SQLite_DB->query('ALTER TABLE `wp...')\n#2 /wordpress/wp-content/plugins/captcha/captcha.php(306): cptch_create_table()\n#3 /wordpress/wp-includes/class-wp-hook.php(324): cptch_plugin_activate(false)\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_captch...', false)\n#7 /internal/eval.php(20): activate_plugin('captcha/captcha...')\n#8 {main}</pre>\n for query ALTER TABLE `wp_cptch_whitelist` ADD UNIQUE(`ip`); made by activate_plugin, do_action('activate_captcha/captcha.php'), WP_Hook->do_action, WP_Hook->apply_filters, cptch_plugin_activate, cptch_create_table, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/c/compartilhe/error.json"}
,
{"message":"Data type expected in MySQL query, unknown token received: settings.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"compartilhe","details":"[17-Apr-2025 04:00:56 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE `wp_cptch_packages` ADD (`settings` LONGTEXT NOT NULL, `user_settings` LONGTEXT NOT NULL, `add_time` DATETIME NOT NULL );</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Data type expected in MySQL query, unknown token received: settings.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/captcha/captcha.php(277): WP_SQLite_DB->query('ALTER TABLE `wp...')\n#2 /wordpress/wp-content/plugins/captcha/captcha.php(306): cptch_create_table()\n#3 /wordpress/wp-includes/class-wp-hook.php(324): cptch_plugin_activate(false)\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_captch...', false)\n#7 /internal/eval.php(20): activate_plugin('captcha/captcha...')\n#8 {main}</pre>\n for query ALTER TABLE `wp_cptch_packages` ADD (`settings` LONGTEXT NOT NULL, `user_settings` LONGTEXT NOT NULL, `add_time` DATETIME NOT NULL ); made by activate_plugin, do_action('activate_captcha/captcha.php'), WP_Hook->do_action, WP_Hook->apply_filters, cptch_plugin_activate, cptch_create_table, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/c/compartilhe/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 duplicate column name: add_time.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"compartilhe","details":"[17-Apr-2025 04:00:56 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE `wp_cptch_whitelist` ADD `add_time` DATETIME;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_cptch_whitelist, `ADD_TIME`, datetime</li>\n<li>Executing: ALTER TABLE wp_cptch_whitelist ADD `add_time` text; | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 duplicate column name: add_time.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/captcha/captcha.php(256): WP_SQLite_DB->query('ALTER TABLE `wp...')\n#2 /wordpress/wp-content/plugins/captcha/captcha.php(355): cptch_create_table()\n#3 /wordpress/wp-content/plugins/captcha/captcha.php(307): cptch_settings()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): cptch_plugin_activate(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_captch...', false)\n#8 /internal/eval.php(20): activate_plugin('captcha/captcha...')\n#9 {main}</pre>\n for query ALTER TABLE `wp_cptch_whitelist` ADD `add_time` DATETIME; made by activate_plugin, do_action('activate_captcha/captcha.php'), WP_Hook->do_action, WP_Hook->apply_filters, cptch_plugin_activate, cptch_settings, cptch_create_table, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/c/compartilhe/error.json"}
,
{"message":"Unknown show type: KEYS FROM.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"compartilhe","details":"[17-Apr-2025 04:00:56 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>SHOW KEYS FROM `wp_cptch_whitelist` WHERE Key_name='ip'</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown show type: KEYS FROM.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/captcha/captcha.php(258): WP_SQLite_DB->query('SHOW KEYS FROM ...')\n#2 /wordpress/wp-content/plugins/captcha/captcha.php(355): cptch_create_table()\n#3 /wordpress/wp-content/plugins/captcha/captcha.php(307): cptch_settings()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): cptch_plugin_activate(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_captch...', false)\n#8 /internal/eval.php(20): activate_plugin('captcha/captcha...')\n#9 {main}</pre>\n for query SHOW KEYS FROM `wp_cptch_whitelist` WHERE Key_name='ip' made by activate_plugin, do_action('activate_captcha/captcha.php'), WP_Hook->do_action, WP_Hook->apply_filters, cptch_plugin_activate, cptch_settings, cptch_create_table, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/c/compartilhe/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 near \")\": syntax error.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"compartilhe","details":"[17-Apr-2025 04:00:56 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE `wp_cptch_whitelist` ADD UNIQUE(`ip`);</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_cptch_whitelist, wp_cptch_whitelist__(, UNIQUE</li>\n<li>Executing: CREATE UNIQUE INDEX &quot;wp_cptch_whitelist__(&quot; ON &quot;wp_cptch_whitelist&quot; (); | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 near \")\": syntax error.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/captcha/captcha.php(259): WP_SQLite_DB->query('ALTER TABLE `wp...')\n#2 /wordpress/wp-content/plugins/captcha/captcha.php(355): cptch_create_table()\n#3 /wordpress/wp-content/plugins/captcha/captcha.php(307): cptch_settings()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): cptch_plugin_activate(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_captch...', false)\n#8 /internal/eval.php(20): activate_plugin('captcha/captcha...')\n#9 {main}</pre>\n for query ALTER TABLE `wp_cptch_whitelist` ADD UNIQUE(`ip`); made by activate_plugin, do_action('activate_captcha/captcha.php'), WP_Hook->do_action, WP_Hook->apply_filters, cptch_plugin_activate, cptch_settings, cptch_create_table, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/c/compartilhe/error.json"}
,
{"message":"Data type expected in MySQL query, unknown token received: settings.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"compartilhe","details":"[17-Apr-2025 04:00:56 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE `wp_cptch_packages` ADD (`settings` LONGTEXT NOT NULL, `user_settings` LONGTEXT NOT NULL, `add_time` DATETIME NOT NULL );</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Data type expected in MySQL query, unknown token received: settings.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/captcha/captcha.php(277): WP_SQLite_DB->query('ALTER TABLE `wp...')\n#2 /wordpress/wp-content/plugins/captcha/captcha.php(355): cptch_create_table()\n#3 /wordpress/wp-content/plugins/captcha/captcha.php(307): cptch_settings()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): cptch_plugin_activate(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_captch...', false)\n#8 /internal/eval.php(20): activate_plugin('captcha/captcha...')\n#9 {main}</pre>\n for query ALTER TABLE `wp_cptch_packages` ADD (`settings` LONGTEXT NOT NULL, `user_settings` LONGTEXT NOT NULL, `add_time` DATETIME NOT NULL ); made by activate_plugin, do_action('activate_captcha/captcha.php'), WP_Hook->do_action, WP_Hook->apply_filters, cptch_plugin_activate, cptch_settings, cptch_create_table, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/c/compartilhe/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 duplicate column name: add_time.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"compartilhe","details":"[17-Apr-2025 04:01:07 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE `wp_cptch_whitelist` ADD `add_time` DATETIME;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_cptch_whitelist, `ADD_TIME`, datetime</li>\n<li>Executing: ALTER TABLE wp_cptch_whitelist ADD `add_time` text; | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 duplicate column name: add_time.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/captcha/captcha.php(256): WP_SQLite_DB->query('ALTER TABLE `wp...')\n#2 /wordpress/wp-content/plugins/captcha/captcha.php(306): cptch_create_table()\n#3 /wordpress/wp-includes/class-wp-hook.php(324): cptch_plugin_activate(false)\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_captch...', false)\n#7 /internal/eval.php(20): activate_plugin('captcha/captcha...')\n#8 {main}</pre>\n for query ALTER TABLE `wp_cptch_whitelist` ADD `add_time` DATETIME; made by activate_plugin, do_action('activate_captcha/captcha.php'), WP_Hook->do_action, WP_Hook->apply_filters, cptch_plugin_activate, cptch_create_table, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/c/compartilhe/error.json"}
,
{"message":"Unknown show type: KEYS FROM.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"compartilhe","details":"[17-Apr-2025 04:01:07 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>SHOW KEYS FROM `wp_cptch_whitelist` WHERE Key_name='ip'</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown show type: KEYS FROM.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/captcha/captcha.php(258): WP_SQLite_DB->query('SHOW KEYS FROM ...')\n#2 /wordpress/wp-content/plugins/captcha/captcha.php(306): cptch_create_table()\n#3 /wordpress/wp-includes/class-wp-hook.php(324): cptch_plugin_activate(false)\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_captch...', false)\n#7 /internal/eval.php(20): activate_plugin('captcha/captcha...')\n#8 {main}</pre>\n for query SHOW KEYS FROM `wp_cptch_whitelist` WHERE Key_name='ip' made by activate_plugin, do_action('activate_captcha/captcha.php'), WP_Hook->do_action, WP_Hook->apply_filters, cptch_plugin_activate, cptch_create_table, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/c/compartilhe/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 near \")\": syntax error.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"compartilhe","details":"[17-Apr-2025 04:01:07 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE `wp_cptch_whitelist` ADD UNIQUE(`ip`);</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_cptch_whitelist, wp_cptch_whitelist__(, UNIQUE</li>\n<li>Executing: CREATE UNIQUE INDEX &quot;wp_cptch_whitelist__(&quot; ON &quot;wp_cptch_whitelist&quot; (); | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 near \")\": syntax error.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/captcha/captcha.php(259): WP_SQLite_DB->query('ALTER TABLE `wp...')\n#2 /wordpress/wp-content/plugins/captcha/captcha.php(306): cptch_create_table()\n#3 /wordpress/wp-includes/class-wp-hook.php(324): cptch_plugin_activate(false)\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_captch...', false)\n#7 /internal/eval.php(20): activate_plugin('captcha/captcha...')\n#8 {main}</pre>\n for query ALTER TABLE `wp_cptch_whitelist` ADD UNIQUE(`ip`); made by activate_plugin, do_action('activate_captcha/captcha.php'), WP_Hook->do_action, WP_Hook->apply_filters, cptch_plugin_activate, cptch_create_table, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/c/compartilhe/error.json"}
,
{"message":"Data type expected in MySQL query, unknown token received: settings.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"compartilhe","details":"[17-Apr-2025 04:01:07 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE `wp_cptch_packages` ADD (`settings` LONGTEXT NOT NULL, `user_settings` LONGTEXT NOT NULL, `add_time` DATETIME NOT NULL );</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Data type expected in MySQL query, unknown token received: settings.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/captcha/captcha.php(277): WP_SQLite_DB->query('ALTER TABLE `wp...')\n#2 /wordpress/wp-content/plugins/captcha/captcha.php(306): cptch_create_table()\n#3 /wordpress/wp-includes/class-wp-hook.php(324): cptch_plugin_activate(false)\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_captch...', false)\n#7 /internal/eval.php(20): activate_plugin('captcha/captcha...')\n#8 {main}</pre>\n for query ALTER TABLE `wp_cptch_packages` ADD (`settings` LONGTEXT NOT NULL, `user_settings` LONGTEXT NOT NULL, `add_time` DATETIME NOT NULL ); made by activate_plugin, do_action('activate_captcha/captcha.php'), WP_Hook->do_action, WP_Hook->apply_filters, cptch_plugin_activate, cptch_create_table, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/c/compartilhe/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 duplicate column name: add_time.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"compartilhe","details":"[17-Apr-2025 04:01:07 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE `wp_cptch_whitelist` ADD `add_time` DATETIME;</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_cptch_whitelist, `ADD_TIME`, datetime</li>\n<li>Executing: ALTER TABLE wp_cptch_whitelist ADD `add_time` text; | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 duplicate column name: add_time.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/captcha/captcha.php(256): WP_SQLite_DB->query('ALTER TABLE `wp...')\n#2 /wordpress/wp-content/plugins/captcha/captcha.php(355): cptch_create_table()\n#3 /wordpress/wp-content/plugins/captcha/captcha.php(307): cptch_settings()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): cptch_plugin_activate(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_captch...', false)\n#8 /internal/eval.php(20): activate_plugin('captcha/captcha...')\n#9 {main}</pre>\n for query ALTER TABLE `wp_cptch_whitelist` ADD `add_time` DATETIME; made by activate_plugin, do_action('activate_captcha/captcha.php'), WP_Hook->do_action, WP_Hook->apply_filters, cptch_plugin_activate, cptch_settings, cptch_create_table, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/c/compartilhe/error.json"}
,
{"message":"Unknown show type: KEYS FROM.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"compartilhe","details":"[17-Apr-2025 04:01:07 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>SHOW KEYS FROM `wp_cptch_whitelist` WHERE Key_name='ip'</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown show type: KEYS FROM.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/captcha/captcha.php(258): WP_SQLite_DB->query('SHOW KEYS FROM ...')\n#2 /wordpress/wp-content/plugins/captcha/captcha.php(355): cptch_create_table()\n#3 /wordpress/wp-content/plugins/captcha/captcha.php(307): cptch_settings()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): cptch_plugin_activate(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_captch...', false)\n#8 /internal/eval.php(20): activate_plugin('captcha/captcha...')\n#9 {main}</pre>\n for query SHOW KEYS FROM `wp_cptch_whitelist` WHERE Key_name='ip' made by activate_plugin, do_action('activate_captcha/captcha.php'), WP_Hook->do_action, WP_Hook->apply_filters, cptch_plugin_activate, cptch_settings, cptch_create_table, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/c/compartilhe/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 near \")\": syntax error.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"compartilhe","details":"[17-Apr-2025 04:01:07 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE `wp_cptch_whitelist` ADD UNIQUE(`ip`);</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_cptch_whitelist, wp_cptch_whitelist__(, UNIQUE</li>\n<li>Executing: CREATE UNIQUE INDEX &quot;wp_cptch_whitelist__(&quot; ON &quot;wp_cptch_whitelist&quot; (); | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 near \")\": syntax error.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/captcha/captcha.php(259): WP_SQLite_DB->query('ALTER TABLE `wp...')\n#2 /wordpress/wp-content/plugins/captcha/captcha.php(355): cptch_create_table()\n#3 /wordpress/wp-content/plugins/captcha/captcha.php(307): cptch_settings()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): cptch_plugin_activate(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_captch...', false)\n#8 /internal/eval.php(20): activate_plugin('captcha/captcha...')\n#9 {main}</pre>\n for query ALTER TABLE `wp_cptch_whitelist` ADD UNIQUE(`ip`); made by activate_plugin, do_action('activate_captcha/captcha.php'), WP_Hook->do_action, WP_Hook->apply_filters, cptch_plugin_activate, cptch_settings, cptch_create_table, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/c/compartilhe/error.json"}
,
{"message":"Data type expected in MySQL query, unknown token received: settings.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"compartilhe","details":"[17-Apr-2025 04:01:07 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE `wp_cptch_packages` ADD (`settings` LONGTEXT NOT NULL, `user_settings` LONGTEXT NOT NULL, `add_time` DATETIME NOT NULL );</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Data type expected in MySQL query, unknown token received: settings.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/captcha/captcha.php(277): WP_SQLite_DB->query('ALTER TABLE `wp...')\n#2 /wordpress/wp-content/plugins/captcha/captcha.php(355): cptch_create_table()\n#3 /wordpress/wp-content/plugins/captcha/captcha.php(307): cptch_settings()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): cptch_plugin_activate(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_captch...', false)\n#8 /internal/eval.php(20): activate_plugin('captcha/captcha...')\n#9 {main}</pre>\n for query ALTER TABLE `wp_cptch_packages` ADD (`settings` LONGTEXT NOT NULL, `user_settings` LONGTEXT NOT NULL, `add_time` DATETIME NOT NULL ); made by activate_plugin, do_action('activate_captcha/captcha.php'), WP_Hook->do_action, WP_Hook->apply_filters, cptch_plugin_activate, cptch_settings, cptch_create_table, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/c/compartilhe/error.json"}
,
{"message":"Uncaught ValueError: max(): Argument #1 ($value) must contain at least one element in /wordpress/wp-content/plugins/cf7-getresponse/lib/tools.php:116","level":"FATAL","type":"PHP","test":"asyncify-boot","plugin":"compartilhe","details":"[17-Apr-2025 04:02:17 UTC] PHP Fatal error:  Uncaught ValueError: max(): Argument #1 ($value) must contain at least one element in /wordpress/wp-content/plugins/cf7-getresponse/lib/tools.php:116\nStack trace:\n#0 /wordpress/wp-content/plugins/cf7-getresponse/lib/tools.php(116): max(Array)\n#1 /wordpress/wp-content/plugins/cf7-getresponse/lib/tools.php(105): vcgr_get_latest_item()\n#2 /wordpress/wp-content/plugins/cf7-getresponse/lib/getresponse.php(22): require_once('/wordpress/wp-c...')\n#3 /wordpress/wp-content/plugins/cf7-getresponse/cf7-extension.php(37): require_once('/wordpress/wp-c...')\n#4 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#5 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('cf7-getresponse...')\n#6 /internal/eval.php(20): activate_plugin('cf7-getresponse...')\n#7 {main}\n  thrown in /wordpress/wp-content/plugins/cf7-getresponse/lib/tools.php on line 116","log":"/logs/plugins/c/compartilhe/error.json"}
,
{"message":"Uncaught ValueError: max(): Argument #1 ($value) must contain at least one element in /wordpress/wp-content/plugins/cf7-getresponse/lib/tools.php:116","level":"FATAL","type":"PHP","test":"asyncify-boot","plugin":"compartilhe","details":"[17-Apr-2025 04:02:25 UTC] PHP Fatal error:  Uncaught ValueError: max(): Argument #1 ($value) must contain at least one element in /wordpress/wp-content/plugins/cf7-getresponse/lib/tools.php:116\nStack trace:\n#0 /wordpress/wp-content/plugins/cf7-getresponse/lib/tools.php(116): max(Array)\n#1 /wordpress/wp-content/plugins/cf7-getresponse/lib/tools.php(105): vcgr_get_latest_item()\n#2 /wordpress/wp-content/plugins/cf7-getresponse/lib/getresponse.php(22): require_once('/wordpress/wp-c...')\n#3 /wordpress/wp-content/plugins/cf7-getresponse/cf7-extension.php(37): require_once('/wordpress/wp-c...')\n#4 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#5 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('cf7-getresponse...')\n#6 /internal/eval.php(20): activate_plugin('cf7-getresponse...')\n#7 {main}\n  thrown in /wordpress/wp-content/plugins/cf7-getresponse/lib/tools.php on line 116","log":"/logs/plugins/c/compartilhe/error.json"}
,
{"message":"Uncaught ValueError: max(): Argument #1 ($value) must contain at least one element in /wordpress/wp-content/plugins/cf7-getresponse/lib/tools.php:116","level":"FATAL","type":"PHP","test":"asyncify-boot","plugin":"compartilhe","details":"[17-Apr-2025 04:02:33 UTC] PHP Fatal error:  Uncaught ValueError: max(): Argument #1 ($value) must contain at least one element in /wordpress/wp-content/plugins/cf7-getresponse/lib/tools.php:116\nStack trace:\n#0 /wordpress/wp-content/plugins/cf7-getresponse/lib/tools.php(116): max(Array)\n#1 /wordpress/wp-content/plugins/cf7-getresponse/lib/tools.php(105): vcgr_get_latest_item()\n#2 /wordpress/wp-content/plugins/cf7-getresponse/lib/getresponse.php(22): require_once('/wordpress/wp-c...')\n#3 /wordpress/wp-content/plugins/cf7-getresponse/cf7-extension.php(37): require_once('/wordpress/wp-c...')\n#4 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#5 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('cf7-getresponse...')\n#6 /internal/eval.php(20): activate_plugin('cf7-getresponse...')\n#7 {main}\n  thrown in /wordpress/wp-content/plugins/cf7-getresponse/lib/tools.php on line 116","log":"/logs/plugins/c/compartilhe/error.json"}
,
{"message":"Uncaught Error: Call to undefined function create_function() in /wordpress/wp-content/plugins/chicago-news-widget-powered-by-everyblock/news.php:84","level":"FATAL","type":"PHP","test":"asyncify-boot","plugin":"compartilhe","details":"[17-Apr-2025 04:03:34 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function create_function() in /wordpress/wp-content/plugins/chicago-news-widget-powered-by-everyblock/news.php:84\nStack trace:\n#0 /wordpress/wp-admin/includes/plugin.php(2387): include_once()\n#1 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('chicago-news-wi...')\n#2 /internal/eval.php(20): activate_plugin('chicago-news-wi...')\n#3 {main}\n  thrown in /wordpress/wp-content/plugins/chicago-news-widget-powered-by-everyblock/news.php on line 84","log":"/logs/plugins/c/compartilhe/error.json"}
,
{"message":"Uncaught Error: Call to undefined function create_function() in /wordpress/wp-content/plugins/chicago-news-widget-powered-by-everyblock/news.php:84","level":"FATAL","type":"PHP","test":"asyncify-boot","plugin":"compartilhe","details":"[17-Apr-2025 04:03:43 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function create_function() in /wordpress/wp-content/plugins/chicago-news-widget-powered-by-everyblock/news.php:84\nStack trace:\n#0 /wordpress/wp-admin/includes/plugin.php(2387): include_once()\n#1 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('chicago-news-wi...')\n#2 /internal/eval.php(20): activate_plugin('chicago-news-wi...')\n#3 {main}\n  thrown in /wordpress/wp-content/plugins/chicago-news-widget-powered-by-everyblock/news.php on line 84","log":"/logs/plugins/c/compartilhe/error.json"}
,
{"message":"Uncaught Error: Call to undefined function create_function() in /wordpress/wp-content/plugins/chicago-news-widget-powered-by-everyblock/news.php:84","level":"FATAL","type":"PHP","test":"asyncify-boot","plugin":"compartilhe","details":"[17-Apr-2025 04:03:50 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function create_function() in /wordpress/wp-content/plugins/chicago-news-widget-powered-by-everyblock/news.php:84\nStack trace:\n#0 /wordpress/wp-admin/includes/plugin.php(2387): include_once()\n#1 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('chicago-news-wi...')\n#2 /internal/eval.php(20): activate_plugin('chicago-news-wi...')\n#3 {main}\n  thrown in /wordpress/wp-content/plugins/chicago-news-widget-powered-by-everyblock/news.php on line 84","log":"/logs/plugins/c/compartilhe/error.json"}
,
{"message":"Uncaught Error: Class \"WC_Payment_Gateway\" not found in /wordpress/wp-content/plugins/coinmall/CoinMall-woocommerce.php:2","level":"FATAL","type":"PHP","test":"asyncify-boot","plugin":"compartilhe","details":"[17-Apr-2025 04:05:26 UTC] PHP Fatal error:  Uncaught Error: Class \"WC_Payment_Gateway\" not found in /wordpress/wp-content/plugins/coinmall/CoinMall-woocommerce.php:2\nStack trace:\n#0 /wordpress/wp-content/plugins/coinmall/coinmall-woocommerce-plugin.php(26): include_once()\n#1 /wordpress/wp-includes/class-wp-hook.php(324): load_coinmall_plugin('')\n#2 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#3 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#4 /wordpress/wp-settings.php(578): do_action('plugins_loaded')\n#5 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#6 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#7 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#8 {main}\n  thrown in /wordpress/wp-content/plugins/coinmall/CoinMall-woocommerce.php on line 2","log":"/logs/plugins/c/compartilhe/error.json"}
,
{"message":"Uncaught Error: Class \"WC_Payment_Gateway\" not found in /wordpress/wp-content/plugins/coinmall/CoinMall-woocommerce.php:2","level":"FATAL","type":"PHP","test":"asyncify-boot","plugin":"compartilhe","details":"[17-Apr-2025 04:05:35 UTC] PHP Fatal error:  Uncaught Error: Class \"WC_Payment_Gateway\" not found in /wordpress/wp-content/plugins/coinmall/CoinMall-woocommerce.php:2\nStack trace:\n#0 /wordpress/wp-content/plugins/coinmall/coinmall-woocommerce-plugin.php(26): include_once()\n#1 /wordpress/wp-includes/class-wp-hook.php(324): load_coinmall_plugin('')\n#2 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#3 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#4 /wordpress/wp-settings.php(578): do_action('plugins_loaded')\n#5 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#6 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#7 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#8 {main}\n  thrown in /wordpress/wp-content/plugins/coinmall/CoinMall-woocommerce.php on line 2","log":"/logs/plugins/c/compartilhe/error.json"}
,
{"message":"Uncaught Error: Class \"WC_Payment_Gateway\" not found in /wordpress/wp-content/plugins/coinmall/CoinMall-woocommerce.php:2","level":"FATAL","type":"PHP","test":"asyncify-boot","plugin":"compartilhe","details":"[17-Apr-2025 04:05:44 UTC] PHP Fatal error:  Uncaught Error: Class \"WC_Payment_Gateway\" not found in /wordpress/wp-content/plugins/coinmall/CoinMall-woocommerce.php:2\nStack trace:\n#0 /wordpress/wp-content/plugins/coinmall/coinmall-woocommerce-plugin.php(26): include_once()\n#1 /wordpress/wp-includes/class-wp-hook.php(324): load_coinmall_plugin('')\n#2 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#3 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#4 /wordpress/wp-settings.php(578): do_action('plugins_loaded')\n#5 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#6 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#7 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#8 {main}\n  thrown in /wordpress/wp-content/plugins/coinmall/CoinMall-woocommerce.php on line 2\n\n    at Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237:15)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)\n    at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {\n  [cause]: Error: Error when executing the blueprint step #1 ({\"step\":\"installPlugin\",\"pluginData\":{\"resource\":\"wordpress.org/plugins\",\"slug\":\"compartilhe\"},\"options\":{\"activate\":true}}) : Could not download \"https://downloads.wordpress.org/plugin/compartilhe.latest-stable.zip\".\n  \t\t\t\tCheck if the URL is correct and the server is reachable.\n  \t\t\t\tIf it is reachable, the server might be blocking the request.\n  \t\t\t\tCheck the browser console and network tabs for more information.\n  \n  \t\t\t\t## Does the console show the error \"No 'Access-Control-Allow-Origin' header\"?\n  \n  \t\t\t\tThis means the server that hosts your file does not allow requests from other sites\n  \t\t\t\t(cross-origin requests, or CORS).\tYou need to move the asset to a server that allows\n  \t\t\t\tcross-origin file downloads. Learn more about CORS at\n  \t\t\t\thttps://developer.mozilla.org/en-US/docs/Web/HTTP/CORS.\n  \n  \t\t\t\tIf your file is on GitHub, load it from \"raw.githubusercontent.com\".\n  \t\t\t\tHere's how to do that:\n  \n  \t\t\t\t1. Start with the original GitHub URL of the file. For example:\n  \t\t\t\thttps://github.com/username/repository/blob/branch/filename.\n  \t\t\t\t2. Replace \"github.com\" with \"raw.githubusercontent.com\".\n  \t\t\t\t3. Remove the \"/blob/\" part of the URL.\n  \n  \t\t\t\tThe resulting URL should look like this:\n  \t\t\t\thttps://raw.githubusercontent.com/username/repository/branch/filename\n  \n  \t\t\t\tError:\n  \t\t\t\tError: Could not download \"https://downloads.wordpress.org/plugin/compartilhe.latest-stable.zip\"\n      at Object.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12449:19)\n      at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n      at async ms (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12552:3)\n      at async Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:227:48)\n      at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)\n      at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {\n    [cause]: Error: Could not download \"https://downloads.wordpress.org/plugin/compartilhe.latest-stable.zip\".\n    \t\t\t\tCheck if the URL is correct and the server is reachable.\n    \t\t\t\tIf it is reachable, the server might be blocking the request.\n    \t\t\t\tCheck the browser console and network tabs for more information.\n    \n    \t\t\t\t## Does the console show the error \"No 'Access-Control-Allow-Origin' header\"?\n    \n    \t\t\t\tThis means the server that hosts your file does not allow requests from other sites\n    \t\t\t\t(cross-origin requests, or CORS).\tYou need to move the asset to a server that allows\n    \t\t\t\tcross-origin file downloads. Learn more about CORS at\n    \t\t\t\thttps://developer.mozilla.org/en-US/docs/Web/HTTP/CORS.\n    \n    \t\t\t\tIf your file is on GitHub, load it from \"raw.githubusercontent.com\".\n    \t\t\t\tHere's how to do that:\n    \n    \t\t\t\t1. Start with the original GitHub URL of the file. For example:\n    \t\t\t\thttps://github.com/username/repository/blob/branch/filename.\n    \t\t\t\t2. Replace \"github.com\" with \"raw.githubusercontent.com\".\n    \t\t\t\t3. Remove the \"/blob/\" part of the URL.\n    \n    \t\t\t\tThe resulting URL should look like this:\n    \t\t\t\thttps://raw.githubusercontent.com/username/repository/branch/filename\n    \n    \t\t\t\tError:\n    \t\t\t\tError: Could not download \"https://downloads.wordpress.org/plugin/compartilhe.latest-stable.zip\"\n        at Ct.resolve (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:1472:13)\n        at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n        at async _.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@php-wasm/util/index.js:45:14)\n  }\n}\n\n","log":"/logs/plugins/c/compartilhe/error.json"}
]
